### ISSUE TITLE
Anyone interested in maintaining svg2tikz?

### ISSUE THREAD

**Issue by kjellmf** [kjellmf/svg2tikz#40]
I have not worked on svg2tikz for a very long time. Is someone interested in taking over the project? Send an email to kjellmf@gmail.com or comment on this issue.

**Comment by rbrito** [comment]
Hi, @kjellmf.

I have potential use for svg2tikz and even though I am not very knowledgeable in any of the languages, I know some python. I can integrate some patches and do some maintenance stuff (say, porting to Python 3, trying to get it packaged on popular Linux distributions like Debian/Ubuntu/etc.).

I just need to get a bit more familiar with the code base. If you are OK with that, then I can send some first pull requests etc.

Thanks for this code in the first place,

Rogério Brito.

**Comment by kjellmf** [comment]
Hi @rbrito  !

Thanks for your interest in maintaining svg2tikz! That's really good news. Take your time and feel free to play with the code base. The code is old, but hopefully it is not that hard to understand. The code quality of my other similar tool, dot2tex, is much worse. 

If you have any questions feel free to ask.

**Comment by aalavandhaann** [comment]
Is this still open? This is an interesting project and yes I know some python, svg, tikz as well. I was about to look into your code to see if there is any support for export image with the output tikz figures. I will start looking into your code base as well. Let us know where we shall begin

**Comment by kjellmf** [comment]
Hi @aalavandhaann 

This issue is still open. If you want to contribute that is great! There are many open issues you can look at. A good place to start is packaging (pypi) and the Inkscape integration. I'm not sure what the current state of Inkscape extensions is.

**Comment by rbrito** [comment]
On December 31, 2019 11:56:42 PM GMT-03:00, #0K Srinivasan Ramachandran <notifications@github.com> wrote:
>Is this still open? This is an interesting project and yes I know some
>python, svg, tikz as well. I was about to look into your code to see if
>there is any support for export image with the output tikz figures. I
>will start looking into your code base as well. Let us know where we
>shall begin

Hi... I wasn't able to find some time to look at the project and I am learning the internals of PDF's right now.

I hope to be able to help with the occasional patch, but I still have to learn some SVG and I am spread too thin among many things.

In other words, feel free to go ahead, as far as I am concerned.

Thanks for caring about this project,

Rogério Brito.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by aalavandhaann** [comment]
I will get on to pypi first. This is something I have done for my other projects. But do you think its time we change the codebase to python3? What do you think? BTW is it okay if we can discuss over email?
Regards,
#0K

**Comment by kjellmf** [comment]
@aalavandhaann Good idea to move the codebase to python3. No need to maintain compatibility with python2 any more. I assume Inkscape has moved to python3 already. 
Feel free to discuss over email.

**Comment by aalavandhaann** [comment]
Hey @kjellmf ,

Actually I finished the Image output for tikz. Also found out that the path nodes are not converted to the tikz correctly. There seems to be some kind of transformation problem. Shapes such as circle, rectangle are exported correctly. But the path node is getting some weird values. Any ideas? 

Regards,

#0K

**Comment by aalavandhaann** [comment]
Hi @kjellmf ,

I can confirm now that it would be a great honor for me to help maintaining this useful plugin for Inkscape. However, I will leave it to your wisdom in making the choice, LOL. 

Regards,
#0K

**Comment by kjellmf** [comment]
I'm closing this issue for now. Thank you to to @aalavandhaann  and @rbrito for responding.

**Comment by kjellmf** [comment]
I'm considering reopening this issue. @aalavandhaann  are you still interested in maintaining svg2tikz?

**Comment by ldevillez** [comment]
I'm interested in maintaining the repo :)
